applications/PoromechanicsApplication/CMakeLists.txt

```diff
@@ -2,6 +2,10 @@ set(CMAKE_INCLUDE_CURRENT_DIR ON)
 
 message("**** configuring KratosPoromechanicsApplication ****")
 
+kratos_add_dependency(${KRATOS_SOURCE_DIR}/applications/StructuralMechanicsApplication)
+kratos_add_dependency(${KRATOS_SOURCE_DIR}/applications/LinearSolversApplication)
+kratos_add_dependency(${KRATOS_SOURCE_DIR}/applications/FluidDynamicsApplication)
```

Comment:
@ipouplana Are we sure the Poromechanics app requires the FluidDynamics App? I guess that someone put it here for a reason (maybe I put it),